content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -5,130 +5,153 @@ Troubleshooting
 
 -  The IoT Box is not connected to the Internet;
 
--  The code is only valid for 5 minutes after the IoT Box has started. It's
-   automatically removed from connected displays when this delay has expired;
+-  The code is only valid for 5 minutes after the IoT Box has started. It's automatically removed
+   from connected displays when this delay has expired;
 
--  The version of the IoT Box image is too old. It should use version 20.06
-   or more recent. If your IoT Box image is from an earlier version, then
-   you will have to reflash the SD card of your IoT Box to update the image
-   (see :doc:`Flashing your SD Card <flash_sdcard>`)
+-  The version of the IoT Box image is too old. It should use version 20.06 or more recent. If the
+   IoT Box image is from an earlier version, then the SD card of the IoT Box will need to be
+   re-flashed to update the image. (see :doc:`Flashing the SD Card <flash_sdcard>`)
 
-If you are not in any of the cases listed above, make sure that the IoT Box
-has correctly started, by checking that a fixed green LED is showing next to
-the micro-USB port.
+If none of the cases listed above correct the issue, then make sure that the IoT Box has correctly
+started, by checking that a fixed green LED is showing next to the micro-USB port.
 
-I've connected my IoT Box but it's not showing in my database
--------------------------------------------------------------
+IoT Box is connected but it's not showing in the database
+---------------------------------------------------------
 
-When you connect an IoT Box to a database, the IoT Box might restart, if
-that is the case, it might take up to one minute before appearing in your
-database. If after some time the IoT is still not showing, make sure that
-your database can be reached from the IoT Box and that your server doesn't
+When an IoT Box connects to a database, the IoT Box might restart, if that is the case, it might
+take up to one minute before appearing in the database. If after some time the IoT is still not
+showing, make sure that the database can be reached from the IoT Box and that the server doesn't
 use a multi-database environment.
 
-My IoT Box is connected to my database, but cannot be reached
--------------------------------------------------------------
+Virtual Windows IoT box not showing on the database
+---------------------------------------------------
+
+In some instances a manual restart of the physical IoT box can also resolve the issue. For the
+Windows virtual IoT box a manual restart of the Odoo server can resolve database connection issues.
+
+To restart the virtual Windows IoT server:
+#. Type "Services" into the :guilabel:`Search Bar`
+#. Select the Services App and scroll down to the :guilabel:`Odoo` Service.
+#. Right click on :guilabel:`Odoo` and select :guilabel:`Start`. This action will manually restart
+the Odoo IoT server.
+
+
+The IoT Box is connected to the Odoo database, but cannot be reached
```

Comment:
I don't really know if we can do this in rst. But the titles are super long so if we want to put a link in Odoo to the documentation (for example, if we can detect that the issue is this one) then it would be more convenient if we have shorter section name.
I don't necessarily mean that we should have title with less text, but we can likely configure custom anchors with shorter name like mentioned at:
https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#explicit-targets

So that instead of having:
https://www.odoo.com/documentation/16.0/applications/productivity/iot/config/troubleshooting.html#my-iot-box-is-connected-to-my-database-but-cannot-be-reached
we can access the same section at:
https://www.odoo.com/documentation/16.0/applications/productivity/iot/config/troubleshooting.html#iot-not-reachable
for instance